src/components/idenfy/IdenfyModal.tsx

```diff
@@ -23,24 +23,21 @@ const IdenfyModal: FunctionComponent<IdenfyModalProps> = ({
   onClose,
 }) => {
   const dispatch = useDispatch();
+  const [shouldRequestProfile, setShouldRequestProfile] = useState(false);
   const [idenfyAuthToken, setIdenfyAuthToken] = useState(
     Cookies.get("idenfyAuthToken")
   );
-
   const { verificationPingStartedAt, profile: { verificationStatus } = {} } =
     useSelector(selectSession);
+  const isVerified = verificationStatus === "Verified";
 
   if (!idenfyAuthToken) {
     dispatch(getIdenfyAuthToken());
   }
 
-  /** Triggers onClose when user is verified and removes verification timer. */
-  if (verificationStatus === "Verified") {
-    const event = new Event("close");
-
+  /** Removes verification timer when verified. */
+  if (isVerified && verificationPingStartedAt) {
     dispatch(removeVerificationTimer());
-
-    onClose(event);
```

Comment:
Removed the modal close event here when the user is verified bc it seems a bit aggressive and we can just let them close the modal, and when they do, they wont be able to open it again bc ✨magic ✨